content/docs/identity-providers/cognito.mdx

```diff
@@ -85,6 +85,14 @@ If you need to make changes after creating your pool, be aware that some setting
 
 You can choose whether to use your own **Domain Name**, or use an AWS-provided one. The AWS-provided domain names are in the format `https://${DOMAIN-PREFIX}.auth.${AWS-REGION}.amazoncognito.com`
 
+### Configure allowed sign-out URLs
+
+Add the special Pomerium sign-out URL to Cognito's **Allowed sign-out URLs** list:
+
+`https://{AUTHENTICATE_DOMAIN}/.pomerium/signed-out`
```

Comment:
I think this should really be `signed_out`, with an underscore rather than a hyphen.

Reply:
@kenjenkins were you able to confirm that `signed_out` is correct?

Reply:
It's `signed_out` in the handler endpoint code here: https://github.com/pomerium/pomerium/blob/c84a251c933dd674211d0a96605e56abcb63fde1/authenticate/handlers.go#L91.

Denis has updated it in in his pending changelog PR as well: https://github.com/pomerium/pomerium/pull/4896/commits/4dd88595f1a42fedc426605564182bc62afa60f2.